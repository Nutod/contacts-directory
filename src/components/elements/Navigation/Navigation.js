import React from "react";
import NavigationItem from "./NavigationItem/NavigationItem";

export default function Navigation() {
	return (
		<ul>
			<NavigationItem />
			<NavigationItem />
			<NavigationItem />
		</ul>
	);
}
