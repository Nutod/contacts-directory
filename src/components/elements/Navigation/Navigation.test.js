import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";
import Navigation from "./Navigation";

Enzyme.configure({
	adapter: new Adapter()
});

describe("<Navigation />", () => {
	const wrapper = shallow(<Navigation />);

	it("Should match snapshot", () => {
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
