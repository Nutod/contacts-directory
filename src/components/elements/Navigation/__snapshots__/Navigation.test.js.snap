// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navigation /> Should match snapshot 1`] = `
<styled.ul>
  <NavigationItem>
    Home
  </NavigationItem>
  <NavigationItem>
    About
  </NavigationItem>
  <NavigationItem>
    Pricing
  </NavigationItem>
  <NavigationItem>
    Contact Us
  </NavigationItem>
</styled.ul>
`;
